<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEditPage</class>
 <widget class="QWidget" name="FileEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="actionView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="add">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveUp">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>/\</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveDown">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>\/</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Action Type:</string>
           </property>
           <property name="buddy">
            <cstring>actionType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="actionType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="findPage">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Find</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QTextEdit" name="find">
                <property name="font">
                 <font>
                  <family>Courier</family>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="editPage">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Edit</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Edit type:</string>
                  </property>
                  <property name="buddy">
                   <cstring>editType</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="editType">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextEdit" name="edit">
                <property name="font">
                 <font>
                  <family>Courier</family>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inlineFindPage">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Inline Find</string>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Find:</string>
                </property>
                <property name="buddy">
                 <cstring>inlineFind</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inlineFind">
                <property name="font">
                 <font>
                  <family>Courier</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inlineEditPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Inline Edit</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Edit Type:</string>
                </property>
                <property name="buddy">
                 <cstring>inlineEditType</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="inlineEditType"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Edit:</string>
                </property>
                <property name="buddy">
                 <cstring>inlineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="inlineEdit">
                <property name="font">
                 <font>
                  <family>Courier</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current File:</string>
       </property>
       <property name="buddy">
        <cstring>currentFile</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="currentFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>currentFile</tabstop>
  <tabstop>actionView</tabstop>
  <tabstop>actionType</tabstop>
  <tabstop>find</tabstop>
  <tabstop>editType</tabstop>
  <tabstop>edit</tabstop>
  <tabstop>inlineFind</tabstop>
  <tabstop>inlineEditType</tabstop>
  <tabstop>inlineEdit</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>moveUp</tabstop>
  <tabstop>moveDown</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
